cmd/influxd/server_integration_test.go

```diff
@@ -177,7 +178,7 @@ func write(t *testing.T, node *Node, data string) {
 
 // query executes the given query against all nodes in the cluster, and verifies no errors occured, and
 // ensures the returned data is as expected
-func query(t *testing.T, nodes Cluster, urlDb, query, expected string) (string, bool) {
+func query(t *testing.T, nodes Cluster, urlDb, query, expected, expectPattern string) (string, bool) {
```

Comment:
Would we also need two versions of `queryAndWait` then?  I wasn't crazy about adding the extra parameter but it seemed like the simplest solution.  If we factor into two funcs, every caller has to do the `if expected != ""` check.  No strong preference here...just don't want to add complication.